src/index.js

```diff
@@ -26,9 +26,9 @@ function crossEnv(args) {
 
 function getCommandArgsAndEnvVars(args) {
   const envVars = getEnvVars()
-  const commandArgs = args.map(commandConvert)
-  const command = getCommand(commandArgs, envVars)
-  return [command, commandArgs, envVars]
+  const commandArgs = args.slice()
+  const command = commandConvert(getCommand(commandArgs, envVars))
+  return [commandConvert(command), commandArgs.map(commandConvert), envVars]
```

Comment:
In addition, we could probably refactor out the whole loop to simplify the code a bit and remove the mutation. 😀